src/commands/publish.ts

```diff
@@ -21,22 +22,39 @@ import { Ignore } from "../context/ignore.ts";
 
 import { getAPIKey } from "../keyfile.ts";
 import { version } from "../version.ts";
+import { setupLog } from "../log.ts";
+
+interface File {
+  fullPath: string;
+  path: string;
+  lstat: Deno.FileInfo;
+}
+
+const nullConfig = {
+  name: "",
+  files: [""],
+};
+
+const nullIgnore = {
+  accepts: [],
+  denies: [],
+};
```

Comment:
This seems not so clean...

Reply:
I know, it's to detect that the function had an error. 
It's not clean at all but we want to avoid at all cost the use of `Deno.exit` anywhere in the code apart from `deps.ts` else it breaks the debug feature. 
II'll listen to you if you have a better solution 🙂